doc/book_zh/examples_zh/sources/basics/custom-transfer.move

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module examples::restricted_transfer {
+    use sui::tx_context::{Self, TxContext};
+    use sui::balance::{Self, Balance};
+    use sui::coin::{Self, Coin};
+    use sui::object::{Self, UID};
+    use sui::transfer;
+    use sui::sui::SUI;
+    
+    /// 错误码 0： 对应所付费用与转移费用不符
```

Comment:
可能需要说明一下 ，代码里面的中文仅仅是翻译需要，实际注释中是不能有中文的，move 暂时不支持UTF-8注释
It may be necessary to note that the Chinese in the code is only for translation, and the actual comments cannot have Chinese, move does not support UTF-8 comments at the moment

Reply:
Thanks for checking, updated in the overview section.